diagnosisApplication/src/main/interface_adapter/diagnosis/DiagnosisState.java

```diff
@@ -8,134 +8,41 @@ public class DiagnosisState {
 
     private int numDiagnoses = 0;
 
-    private  Integer diagnosis1Ranking = null;
-    private  String diagnosis1Name = null;
-    private  Integer diagnosis1Accuracy = null;
-    private  String diagnosis1Icd = null;
-    private  String diagnosis1ProfName = null;
-    private List<String> diagnosis1Specializations = null;
+    private HashMap<String, Object> diagnosis1;
+    private HashMap<String, Object> diagnosis2;
+    private HashMap<String, Object> diagnosis3;
 
-    private  Integer diagnosis2Ranking = null;
-    private  String diagnosis2Name = null;
-    private  Integer diagnosis2Accuracy = null;
-    private  String diagnosis2Icd = null;
-    private  String diagnosis2ProfName = null;
-    private List<String> diagnosis2Specializations = null;
-
-    private  Integer diagnosis3Ranking = null;
-    private  String diagnosis3Name = null;
-    private  Integer diagnosis3Accuracy = null;
-    private  String diagnosis3Icd = null;
```

Comment:
Initialization of these instance variables in no longer required since it will be stored in a hash map.